torch/_refs/__init__.py

```diff
@@ -411,7 +410,7 @@ def _ref(
     if aten_op is infer_aten_op:
         aten_op = getattr(torch.ops.aten, prim.__name__.split(".")[0])
     if aten_op is not None:
-        register_decomposition(aten_op, register_meta=register_meta)(_ref)
+        register_decomposition(aten_op, disable_meta=disable_meta)(_ref)
```

Comment:
primtorch refs go into the decomposition table too? That's interesting -- are there cases where we have multiple decompositions registered for an op?

Reply:
I'd expect the primtorch decomp to be a lot slower due to needing to model things like type promotion, does that matter in practice?

Reply:
> It seems better to trade performance for correctness

I agree that it's better to trade performance for correctness.